views/blocks/payment_other_with_logo.tpl

```diff
@@ -0,0 +1,55 @@
+[{*
+Uses the content of original OXID payment_other.tpl
+Rendering logo was needed but could not be achieved by extending the block
+*}]
+<dl>
+    <dt>
+        [{assign var="oShopConf" value=$oViewConf->getConfig()}]
+        <input id="payment_[{$sPaymentID}]" type="radio" name="paymentid" value="[{$sPaymentID}]" [{if $oView->getCheckedPaymentId() == $paymentmethod->oxpayments__oxid->value}]checked[{/if}]>
+        [{* Logo added *}]
+        <img src="[{$oShopConf->getShopUrl()}]modules/wirecard/paymentgateway/out/img/[{ $paymentmethod->oxpayments__wdoxidee_logo->value }]">
+        <label for="payment_[{$sPaymentID}]"><b>[{$paymentmethod->oxpayments__oxdesc->value}]</b></label>
+    </dt>
+    <dd class="[{if $oView->getCheckedPaymentId() == $paymentmethod->oxpayments__oxid->value}]activePayment[{/if}]">
+        [{if $paymentmethod->getPrice()}]
+            [{assign var="oPaymentPrice" value=$paymentmethod->getPrice() }]
+            [{if $oViewConf->isFunctionalityEnabled('blShowVATForPayCharge') }]
+                [{strip}]
+                    ([{oxprice price=$oPaymentPrice->getNettoPrice() currency=$currency}]
+                    [{if $oPaymentPrice->getVatValue() > 0}]
+                        [{oxmultilang ident="PLUS_VAT"}] [{oxprice price=$oPaymentPrice->getVatValue() currency=$currency}]
+                    [{/if}])
+                [{/strip}]
+            [{else}]
+                ([{oxprice price=$oPaymentPrice->getBruttoPrice() currency=$currency}])
+            [{/if}]
+        [{/if}]
+
+        [{foreach from=$paymentmethod->getDynValues() item=value name=PaymentDynValues}]
+            <div class="form-group">
+                <label class="control-label col-lg-3" for="[{$sPaymentID}]_[{$smarty.foreach.PaymentDynValues.iteration}]">[{$value->name}]</label>
+                <div class="col-lg-9">
+                    <input id="[{$sPaymentID}]_[{$smarty.foreach.PaymentDynValues.iteration}]" type="text" class="form-control textbox" size="20" maxlength="64" name="dynvalue[[{$value->name}]]" value="[{$value->value}]">
+                </div>
+            </div>
+        [{/foreach}]
+
+        <div class="clearfix"></div>
+
+        [{if $paymentmethod->oxpayments__oxid == "wdcreditcard"}]
```

Comment:
Is this really supposed to be displayed or is this a debugging leftover?

Reply:
It should be displayed here, as stated in the ticket.